nle/agent/core/vtrace.py

```diff
@@ -125,10 +125,7 @@ def from_importance_weights(
         vs = torch.add(vs_minus_v_xs, values)
 
         # Advantage for policy gradient.
-        broadcasted_bootstrap_values = torch.ones_like(vs[0]) * bootstrap_value
-        vs_t_plus_1 = torch.cat(
-            [vs[1:], broadcasted_bootstrap_values.unsqueeze(0)], dim=0
-        )
+        vs_t_plus_1 = torch.cat([vs[1:], torch.unsqueeze(bootstrap_value, 0)], dim=0)
```

Comment:
What is the rationale for changing this file? Are you sure the broadcasting isn't needed?
Just wondering as this file is not original, but copied from elsewhere.

Reply:
https://github.com/facebookresearch/nle/blob/0424d298f52f21e4e3f7dda09381051ce8359e30/nle/agent/core/vtrace.py#L128

Pulled straight out of our `neurips2020release` branch

Reply:
Let's leave this file as-is.